src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -396,6 +413,29 @@ private static String createGCSBucketWithMultipleFiles(String folderPath) throws
     return bucketName;
   }
 
+  private static String createMultipleGCSBucketsWithMultipleFiles(String folderPath)
```

Comment:
can we make this method more generic and add no of buckets to be created as a param?

Reply:
yes. will edit the function and also add some wildcard delete test